### ISSUE TITLE
Migration of contribution module to kotlin

### ISSUE THREAD

**Issue by Saifuddin53** [commons-app/apps-android-commons#6011]
Duplicate of #5968

**Comment by neeldoshii** [comment]
Hi @Saifuddin53, if you don't mind can you pick other module. I have already started working on this module #5968.

**Comment by Saifuddin53** [comment]
Hello @neeldoshii , apologies i overlooked your issue

**Comment by neeldoshii** [comment]
Hi @Saifuddin53 👋, if u wanna take this issue than go ahead.

**Comment by Saifuddin53** [comment]
Hello @neeldoshii , will be on these after #6017
